packages/components/src/misc/NumericInput.component.svelte

```diff
@@ -0,0 +1,95 @@
+<script lang="ts">
+  export let value = 0;
+  export let min = 0;
+  export let max = 0;
+  export let callback = (value: number) => {};
+
+  function changeGroup(offset: number) {
+    value = Math.min(Math.max(value + offset, min), max);

```

Comment:
We can create `clamp` utility function to help with that, it will help with readability. (FYI a lot of these helper functions are provided from functional libraries like [ramda](https://ramdajs.com/) we can discuss if it makes sense to add as dependency).